plugin.cpp

```diff
@@ -231,13 +232,15 @@ inline access_info unwrap_access( cexpr_t* expr, bool is_assignee = false )
 			res.shift_value = std::get<2>( *it );
 		}
 		// handle upper bit access that's transformed to a sign bit comparison.
-		// e.g. `x < 0`
-		else if ( expr->op == cot_slt )
+		// e.g. `x < 0`, `x <= 0`, `x > 0` and `x >= 0`
+		else if ( expr->op == cot_slt || expr->op == cot_sle || expr->op == cot_sgt || expr->op == cot_sge )
```

Comment:
I think only `<` and `>=` make sense. I'm not sure if it'd be sane to do anything with `<=` and `>`.